### ISSUE TITLE
Gradient accumulation strategy when fine-tuning 1-64 views

### ISSUE THREAD

**Issue by photosartd** [CUT3R/CUT3R#80]
Hey, I have a question regarding the fine-tuning setup for 64 views. In the [config](https://github.com/CUT3R/CUT3R/blob/main/config/dpt_512_vary_4_64.yaml), I see:

```
batch_size: 4  
accum_iter: 4
```

At the same time, in `loss_of_one_batch_tbptt`, the optimizer step is performed after each of the last 4 batches (i.e., chunks). This means that in a distributed training setup, each GPU updates its model weights locally after each of those chunks.

However, since we have `accelerator.accumulate` with `accum_iter = 4`, gradient synchronization (via all-reduce) only happens every 4th batch. As a result, the model parameters across GPUs would start to diverge after the first step and never become identical again.

Do I understand this correctly and is this actually what happens?
